.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [ ] Please, remove the old commented code found in:

- Details.js
- Item.js